.circleci/config.yml

```diff
@@ -454,43 +472,43 @@ workflows:
       - build-drupal9-govcms:
           filters:
             branches:
-              ignore: /^content\/.*/
+              only: /develop|release\/.*/
             tags:
               only: /.*/
       - build-drupal9-minimal:
           filters:
             branches:
-              ignore: /^content\/.*/
+              only: /develop|release\/.*/
             tags:
               only: /.*/
       - build-drupal9-minimal-default:
           filters:
             branches:
-              ignore: /^content\/.*/
+              only: /^release\/.*/
             tags:
               only: /.*/
       - build-drupal9-minimal-corporate:
           filters:
             branches:
-              ignore: /^content\/.*/
+              only: /^release\/.*/
             tags:
               only: /.*/
       - build-drupal9-minimal-highereducation:
           filters:
             branches:
-              ignore: /^content\/.*/
+              only: /^release\/.*/
```

Comment:
Ticket says you should build the content profiles for develop? Did you mean release?

Reply:
@richardgaunt 
Ticket description is wrong (it was a "thought" only).

We have content profiles that are built in CI (as a separate job per profile) and, once passed and merged to `develop` - an additional job mirrors code into per-profile branches (`content-corporate`, `content-government` etc) which then triggers a deployment in Lagoon (like https://nginx-php.content-highereducation.civictheme-source.lagoon.salsa.hosting/) . This allows us to QA content profiles before they reach the release.

But running these builds for every PR and even for every merge to `develop` appears to be expensive (budget and time-wise).

So, the change here is to:
 - In CI - not build content profiles in PR or on merge in `develop` and only build them on `release/` branches. This should catch any issues in code.
 - In Lagoon - still build them on each merge to `develop` - this still allows us to QA content profiles before they reach the release.

This is not ideal of course(both CI and Lagoon should have these running), but practically deploying to Lagoon is ok without Ci passing for these profiles as these deployments (https://nginx-php.content-highereducation.civictheme-source.lagoon.salsa.hosting/) are for QA only.